lib/traject/macros/custom.rb

```diff
@@ -226,6 +226,13 @@ def hathi_to_hash(ht_format)
         end
       end
 
+      def exclude_locations
+        lambda do |_record, accumulator|
+          accumulator.reject! { |value| ConfigSettings.location_excludes.include?(value) }
+          accumulator.reject!(&:empty?)
```

Comment:
is this the same as `accumulator.compact`? Just eliminating empty items?

Reply:
ah yes most likely, I just forget about that method :) I will check and change it so it is consistent with the rest

Reply:
I do love me a pretzel colon operator - but yeah I think if it's the same as compact that probably makes more sense (being nitpicky I know sorry)